lnrpc/rpc.proto

```diff
@@ -2028,6 +2028,24 @@ message PendingChannelsResponse {
 
         /// Hash of the remote pending version of the commitment tx.
         string remote_pending_txid = 3;
+
+        /*
+        The amount in satoshis calculated to be paid in fees for the local
+        commitment.
+        */
+        uint64 local_commit_fee = 4;
```

Comment:
nit: `_sat` suffix for all of these?

Reply:
I think having these suffixes may make it harder to adopt msat (if we ever do that :)) as we'll need to deprecate a bunch of fields and introduce new ones. wdyt @guggero ?

Reply:
Or go straight to msat with these? Feels conuterintuitive for values that come straight off a commitment, but if all msat fields is where we want to be (is it?), could just skip ahead to that.

Reply:
There will probably never be msats for this, as it's chain related. But a user wouldn't necessarily know this so the `_sat` suffix would be to make sure everyone understands what this value contains. I'm also fine with an explicit mention in the comment only. But I wouldn't convert it to msat, that wouldn't make sense really.

Reply:
Went with `_sat`, looks like we use it in a few other places :) Agree that msat would be odd here.